### PR TITLE
EDSC-2136: Initial load state styles

### PR DESCRIPTION
**Pull request by trevorlang**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -71,13 +72,15 @@
     "serverless-webpack": "^5.2.0",
     "style-loader": "^0.23.1",
     "webpack": "^4.24.0",
-    "webpack-bundle-analyzer": "^3.0.3",
+    "webpack-bundle-analyzer": "^3.3.2",
     "webpack-cli": "^3.2.3",
     "webpack-dev-server": "^3.1.10",
-    "webpack-merge": "^4.2.1"
+    "webpack-merge": "^4.2.1",
+    "why-did-you-update": "^1.0.6"
```
Comment (trevorlang) [inline package.json]: dev dependancy

File: static/src/js/containers/FacetsContainer/FacetsContainer.js [hunk 2]
```diff
@@ -11,10 +11,13 @@ const mapDispatchToProps = dispatch => ({
   onChangeFeatureFacet: (e, facetLinkInfo) => dispatch(actions.changeFeatureFacet(e, facetLinkInfo))
 })
 
-const mapStateToProps = state => ({
-  facets: state.entities.facets,
-  featureFacets: state.facetsParams.feature
-})
+const mapStateToProps = (state) => {
```
Comment (trevorlang) [inline static/src/js/containers/FacetsContainer/FacetsContainer.js]: Unintentional

File: static/src/js/containers/SidebarContainer/SidebarContainer.js [hunk 3]
```diff
@@ -7,13 +7,15 @@ import Header from './Header/Header'
...
 
-const SidebarContainer = () => (
-  <section className="sidebar">
-    <Header />
-    <SidebarSection sectionTitle="Browse Collections">
-      <ConnectedFacetsContainer />
-    </SidebarSection>
-  </section>
-)
+const SidebarContainer = () => {
```
Comment (trevorlang) [inline static/src/js/containers/SidebarContainer/SidebarContainer.js]: Unintentional

File: static/src/js/containers/MasterOverlayPanelContainer/MasterOverlayPanelContainer.js [hunk 4]
```diff
@@ -1,25 +1,80 @@
 import React from 'react'
 import { connect } from 'react-redux'
 import PropTypes from 'prop-types'
+// import { pure } from 'recompose'
```
Comment (trevorlang) [inline static/src/js/containers/MasterOverlayPanelContainer/MasterOverlayPanelContainer.js]: Remove comment

File: static/src/js/components/CollectionResults/CollectionResultsBody.js [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+    onFocusedCollectionChange(collectionId)
+  }
+
+  render() {
+    const {
+      collections,
+      location
+    } = this.props
+
+    // const collectionHits = collections.hits
```
Comment (macrouch) [inline static/src/js/components/CollectionResults/CollectionResultsBody.js]: Unused

File: static/src/js/components/SpatialDisplay/SpatialSelectionDropdown.js [hunk 6]
```diff
@@ -0,0 +1,71 @@
+import React, { PureComponent } from 'react'
+// import PropTypes from 'prop-types'
```
Comment (macrouch) [inline static/src/js/components/SpatialDisplay/SpatialSelectionDropdown.js]: Unused

**Pull request merged**